misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JClass.java

```diff
@@ -39,68 +38,66 @@
 
 /**
  * Class for storing details pertaining to a specific Java class used for Ballerina bridge code generation.
+ *
+ * @since 1.2.0
  */
 public class JClass {
 
-    public String shortClassName;
-    public String packageName;
-
     private String prefix;
```

Comment:
This variable is assigned but never accessed

Reply:
This field is used in the handlebars template.